src/theme.rs

```diff
@@ -24,6 +24,9 @@ pub enum BinaryColorTheme {
 
     /// An on/off OLED-like display with a dark blue background and light blue pixels
     OledBlue,
+
+    /// Custom binary color theme/mapping
+    Custom(Rgb888, Rgb888),
```

Comment:
I would prefer a struct here to give the two mappings a name:
```rust
Custom {
    color_off: Rgb888,
    color_on: Rgb888,
}
```

Reply:
Hm... ok.. I thnk that's the semantics the constructor `pub const fn custom(color_off: Rgb888, color_on: Rgb888) -> BinaryColorTheme {...}` would communicate to the API user. Wouldn't adding an additional type for this simple thing be a bit overengineered?

Reply:
Sorry, I think I wasn't clear enough. I just meant to change the enum variant from a tuple into a struct and not to add a new type, which I agree would be over engineered.

> I thnk that's the semantics the constructor `pub const fn custom(color_off: Rgb888, color_on: Rgb888) -> BinaryColorTheme {...}` would communicate to the API user.

I don't think the constructor is as discoverable as the enum variants themselves, because they are listed first in the docs and a user might not expect that one of the variants has a constructor.